client/src/modules/journal/modals/editTransaction.modal.js

```diff
@@ -342,7 +344,8 @@ function JournalEditTransactionController(
             };
 
             Modal.close(deleteTransactionResult);
-          });
+          })
+          .catch(Notify.handleError);
```

Comment:
Can you just log into a `<span class="text-danger"> SOMETHING BROKE!!!</span>`?

Reply:
I have created a customised one : 

![image](https://user-images.githubusercontent.com/5445251/79020956-355d0c00-7b72-11ea-93aa-028a4290ffa2.png)